File: alphabet_2.py
# Created by : Sidney Kang
# Created on : 23 Oct. 2017
# Created for : ICS3UR
# Daily Assignment - Unit3-09
# This program displays a nested loop program that prints capital letters
#   from A - Z and lower case

for code in range(65,91):
    for code_2 in range(97,123):     
           print(str(unichr(code)) + " => " + str(unichr(code_2)))

File: alphabet2.java
/****************************************************************************
 *
 * Created on : 30 Oct. 2017
 * Created for : ICS3UR
 * daily Assignment - Unit 3-09
 * This program displays a nested loop program that prints capital letters
 *   from A - Z and lower case
 *
 ****************************************************************************/
 
public class alphabet 
{
     public static void main(String[] args)
    {
         for (int ascii = 65; ascii <= 90; ascii++)
              for (int ascii2 = 97; ascii2 <= 122; ascii2++)
        {
              {  
                   char newCapitalCharacter = (char) ascii;
                   char newLowerCharacter = (char) ascii2;
                   System.out.println(newCapitalCharacter + " => " +  newLowerCharacter);
              }   
         }
     }
}
